tests/openvino/tools/calibrate.py

```diff
@@ -212,9 +212,6 @@ def quantize_model(xml_path, bin_path, accuracy_checcker_config,
     model_evaluator = create_model_evaluator(accuracy_checcker_config)
     model_evaluator.load_network([{'model': ov_model}])
     model_evaluator.select_dataset('')
-    # workaround to minimize time of estimation len of dataset
-    subset_size = quantization_parameters.get('subset_size', 300)
```

Comment:
In case of dropping this, the quantization time would be increased, because the statistics will be collected from the whole dataset. Am I right?